hubbot

```diff
@@ -415,6 +431,9 @@ def expire_work_dirs(github_visible_commits):
             if not dry_run:
```

Comment:
I know this is old code, but i would explicitly put a check that the directory being deleted is a subdirectory of the work directory or a relative directory, and not an absolute directory.